WcaOnRails/app/helpers/application_helper.rb

```diff
@@ -126,4 +126,10 @@ def wca_date_range(from_date, to_date, options={})
     options[:separator] = '-'
     date_range(from_date, to_date, options)
   end
+
+  def alert(type, content=nil, note: false, &block)
+    content = capture(&block) if block_given?
+    content.prepend("<strong>Note:</strong> ") if note
```

Comment:
I think you should call `.html_safe` on this string. SafeBuffer seems to do the correct thing here when appending a SafeBuffer and a string:

```ruby
irb(main):007:0> ("<strong>note:</strong>".html_safe + "YIKES<")
=> "<strong>note:</strong>YIKES&lt;"
```